docker-compose.yml

```diff
@@ -0,0 +1,62 @@
+version: "3.4"
+
+services:
+  server:
+    build:
+      context: ./backend
+      dockerfile: Dockerfile
+    volumes:
+      - ./backend/:/backend/
+    entrypoint: ./docker-start.sh
+    container_name: taia-x-backend
+    env_file:
+      - ./backend/.env
+    ports:
+      - 5678:5678
+      - 8000:8000
+    depends_on:
+      - db
+      - ipfs
+
+  frontend:
+    build:
+      context: ./frontend
+      dockerfile: Dockerfile
+    volumes:
+      - ~/.composer-docker/cache:/root/.composer/cache:delegated
+      - ./frontend/:/opt/taia-x/frontend:rw,delegated
+    container_name: taia-x-frontend
+    environment:
+      - CHOKIDAR_INTERVAL=1000
+    ports:
+      - 8080:8080
+    depends_on:
+      - db
+      - ipfs
```

Comment:
@amrap030, I think this can be removed. The front-end container does not need to wait for the db and ipfs to start

Reply:
Yes, looks good, thank you.